### ISSUE TITLE
Input PIV Parameter issue

### ISSUE THREAD

**Issue by Zainghgh** [mI-PIV/app#310]
mI-PIV app beta version 14 is not allowing us to set input PIV parameters in both default settings as well as in advanced settings.Due to this we are unable to move to the next step.

**Comment by Zainghgh** [comment]
when I set default parameters, I don't see the option to ok it.And the same
problem comes in advance parameter settings when I set it, I don't see the
option to ok it too.I think we don't see the entire window of the PIV
parameter dialogue which is why there is no OK button.

**Comment by kp-121** [comment]
Ah, okay I'll check it out. Thank you!

**Comment by Kevin-Jay-Roberts21** [comment]
I'll take a look at this as well. It may be that we're opening up the popup with a certain pixel size. So on my phone it may look fine but on others it may be too big or small.

**Comment by Kevin-Jay-Roberts21** [comment]
@Zainghgh what phone are you using? Are you still getting this issue? I am using a Google Pixel 4 phone and I'm not seeing any problems. I was wondering if certain types of phones run into this issue.

**Comment by jack-elliott** [comment]
On a Samsung A51 the advanced parameters popup doesn't fit in the screen if
that example helps

**Comment by Zainghgh** [comment]
Hello Kevin,

I am using LG V40 ThinQ phone.Yes still I am facing same issue PIV
Parameters Dialog popup doesn't fit in the screen and we are unable to set
PIV Parameters.

Thanks
Zain

**Comment by Kevin-Jay-Roberts21** [comment]
I was able to test this issue on the LG V40 ThinQ and the Google Pixel 4 phones. The issue originated from trying to increase the width of the popup to make sure the lightbulbs (now information icons) weren't cut off. Apparently, getWindow().getAttributes.width must be set differently for various phones. Using DisplayMetrics, I was able to grab the phones current width, then set a small buffer (35 pixels), to ensure the information icons weren't cut off. 

Commit 3a1bae8 fixes this issue.
